src/components/editor-page/editor-pane/hooks/use-handle-upload.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { uploadFile } from '../../../../api/media'
+import { getGlobalState } from '../../../../redux'
+import { supportedMimeTypes } from '../../../common/upload-image-mimetypes'
+import { t } from 'i18next'
```

Comment:
Does this use the configured i18next instance?

Reply:
Yes! `i18next` is independent from React once it's configured.